### PR TITLE
Include artifact size in logs

### LINKED ISSUE #60: Include artifacts size in the logs
In case an artifact is larger then 1Mb. Right now en error can look like:

```
Uploading 1 artifacts for /tmp/cirrus-ci-build/build/spicy-*.tar.gz
Error from upload stream: rpc error: code = FailedPrecondition desc = Bytes size limit reached!
```

Which caused by the artifact allegedly be more than 1G. Will be nice to log sizes of large artifacts.

Related to #721

### PR DESCRIPTION
**Pull request by edigaryev**
Resolves #60.

### CODE REVIEW

File: internal/executor/artifacts.go [hunk 1]
```diff
@@ -121,10 +122,18 @@ func uploadArtifactsAndParseAnnotations(
...
-			if info, err := os.Stat(artifactPath); err == nil && info.IsDir() {
+			info, err := os.Stat(artifactPath)
+
+			if err == nil && info.IsDir() {
 				continue
 			}
 
+			if err == nil && info.Size() > 1 * humanize.MByte {
+				humanFriendlySize := humanize.Bytes(uint64(info.Size()))
+				logUploader.Write([]byte(fmt.Sprintf("Uploading a quite hefty artifact '%s' of size %s",
```
Comment (fkorotkov) [inline internal/executor/artifacts.go:133]: Let's do 100Mb as a definition  for hefty. Or just log any sizes more then 10Mb.

Comment (fkorotkov) [reply internal/executor/artifacts.go:133]: Or just change the message to not include hefty. 🤔

Comment (edigaryev) [reply internal/executor/artifacts.go:133]: It seemed strange to only log specific artifact uploads and skip others without an explanation (a user might think that some uploads were not done for some reason), so I've added this "hefty" thing.

See https://github.com/cirruslabs/cirrus-ci-agent/pull/62/commits/70a0537cc896d9d98995bc5fe9c2395166dcb257.

**Review by fkorotkov — APPROVED** [review]
IMO 100Mb is hefty but nowadays there are web pages more than 1MB 😅

**Pull request merged**